plugins/woocommerce-admin/client/products/utils/get-product-status.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { PartialProduct } from '@woocommerce/data';
+
+/**
+ * Labels for product statuses.
+ */
+export const PRODUCT_STATUS_KEYS = {
+	unsaved: 'unsaved',
+	draft: 'draft',
+	instock: 'instock',
+	outofstock: 'outofstock',
+};
```

Comment:
Instead of making this an object, should we make this a TypeScript enum? 
```
enum ProductStatusKeys {
  unsaved = 'unsaved',
...
}
```
That way we could pass the specific status types from `getProductStatus` instead of just `string`

Reply:
Good call!  Updated!